deploying/deploying.md

```diff
@@ -45,11 +45,11 @@ Deployment is a fancy term for getting your website on the web. After building o
 
 Now we'll work on deploying your app to Heroku.
 
-We'll be combining your frontend (create-react-app) with your backend (express) and deploying it to Heroku. Your frontend contains "static" JavaScript files -- when you deploy to Heroku, Heroku turns your whole React app into a couple of static files that it will serve to the browser. No matter what data you have in the database, these files will always be the same.
+We'll be combining your front-end (create-react-app) with your back-end (express) and deploying it to Heroku. Your front-end contains "static" JavaScript files -- when you deploy to Heroku, it turns your whole React app into a couple of static files that it will serve to the browser. No matter what data you have in the database, these files will always be the same.
 
 Your backend, on the other hand, is dynamic -- when you make an API request, the backend runs javascript code to do things like reading and writing to a database. Unlike the React app, which always serves the same files to the browser, the backend will serve different information to the browser depending on what's in the database. We're going to combine your dynamic code (express), with your static code (create-react-app).
 
-1. cd into the React app you created and move _everything_ into a new directory named `client`:
+1. cd into the React app and create 2 new directories `client` and `server` you created and move _everything_ into the new directory `client`:
```

Comment:
Thank you for adding that, but could you rephrase the sentence now?  It doesn't make sense anymore.